### ISSUE TITLE
its get farsi languge we need to get more lang

### ISSUE THREAD

**Issue by m0shendy** [DaveOff/SubscenePotplayer#2]
Dear coder 
Problem that needs to be solved. add more languages like arabic , fix a bug
Why solving this problem is important: .more users will be able to use the product.

**Comment by Redruby110** [comment]
yes I think English Persian Indonesian and Arabic,could be added to the Extension.
But thanks for Starting.
